control-plane/pkg/reconciler/broker/broker.go

```diff
@@ -300,26 +305,23 @@ func (r *Reconciler) ConfigMapUpdated(ctx context.Context) func(configMap *corev
 func (r *Reconciler) SetBootstrapServers(servers string) error {
```

Comment:
This function doesn't need to return error anymore

Reply:
Thank you, fixed!